ci/gpu/build.sh

```diff
@@ -95,3 +95,6 @@ conda install -c conda-forge -y feather-format
 logger "Python py.test for cuDF..."
 cd $WORKSPACE/python
 py.test --cache-clear --junitxml=${WORKSPACE}/junit-cudf.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/cudf-coverage.xml --cov-report term
+
+logger "Upload CodeCov results..."
+curl -s https://codecov.io/bash | bash -s -- -c -F aFlag -f ${WORKSPACE}/cudf-coverage.xml
```

Comment:
I don't feel too strongly about this, but there seems to be many others that have problems with "curl bash piping". I know CodeCov documents this as the way to upload results, but they also released a Python package (available from both `pip` and `conda`) so you don't have to do a curl|bash

https://github.com/codecov/codecov-python